File: app/src/main/java/cc/smartcash/wallet/Models/ExplorerApiTxs.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

@JsonIgnoreProperties(ignoreUnknown = true)
public class ExplorerApiTxs implements Serializable {

    @JsonProperty("Time")
    private String time;
    @JsonProperty("Txid")
    private String txId;

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getTxId() {
        return txId;
    }

    public void setTxId(String txId) {
        this.txId = txId;
    }


}

File: app/src/main/java/cc/smartcash/wallet/Models/ExplorerApiTxs.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class ExplorerApiTxs : Serializable {

    @JsonProperty("Time")
    var time: String? = null

    @JsonProperty("Txid")
    var txId: String? = null


}




